src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,61 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React from 'react';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+interface Props {
+  filter: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  setSearchQuery: (query: string) => void;
+  searchQuery: string;
+  clearSearch: () => void;
+}
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+export const TodoFilter: React.FC<Props> = (
+  {
+    filter, setSearchQuery, searchQuery, clearSearch,
+  },
+) => {
+  return (
+    <form className="field has-addons">
+      <p className="control">
+        <span
+          className="select"
+        >
+          <select
+            data-cy="statusSelect"
+            onChange={filter}
+          >
+            <option value="all">All</option>
+            <option value="active">Active</option>
+            <option value="completed">Completed</option>
+          </select>
```

Comment:
can we create a const or enum with our statuses and use it in all places in our application when we need status?